_data/students.yml

```diff
@@ -546,7 +546,7 @@
   twitter: Pro_MYScrub
   bio: Can you differentiate the useful and the useless?
 
-- name: Yee Jia Shuen
+- name: Yee J. S.
```

Comment:
She asked me to change it, but if you need I could just tell her to change it herself